lib/registrant.rb

```diff
@@ -0,0 +1,41 @@
+require 'pry'
+class Registrant
+    attr_reader :name, :age, :permit, :licence_data, :written, :licence, :renewed 
```

Comment:
We only want attr_readers defined for instance variables that we want to expose methods for. So we only want `attr_reader :name, :age, :license_data`. All the other methods are defined below so they should not go in the attr_reader.